docs/data/data-grid/cell-selection/SelectionApiNoSnap.js

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import ApiDocs from 'docsx/src/modules/components/ApiDocs';
+import premiumApi from 'docsx/pages/x/api/data-grid/grid-cell-selection-api.json';
+
+export default function SelectionApiNoSnap() {
```

Comment:
Maybe rename it to `CellSelectionApiNoSnap` to avoid confusion when the demo fails?